steps/move-or-copy-files/0.0.2/step.yml

```diff
@@ -0,0 +1,49 @@
+title: Move or Copy Files
+summary: |
+  Moves or copies files
+description: |
+  This Step copies or moves a single file to the desired target location. It can also be used to rename files.
+website: https://www.datenstrom.online
```

Comment:
please use a step related website URL, where the users can learn more about the step